src/hera/shared/_global_config.py

```diff
@@ -99,6 +99,16 @@ def token(self, t: Union[Optional[str], TokenGenerator, Callable[[], Optional[st
         """Sets the Argo Workflows token at a global level so services can use it."""
         self._token = t
 
+    @property
+    def client_certs(self) -> Optional[Tuple[str, str]]:
+        return self._client_certs
+
+    @client_certs.setter
+    def client_certs(self, certs: Tuple[str, str]) -> None:
```

Comment:
Would it be beneficial for this to have the signature
```suggestion
    def client_certs(self, certs: Tuple[str, str] | Callable(None, Tuple[str, str])) -> None:
```

I am thinking whether it's possible for users to "fetch" the cert/key files using some client logic, store those on local, then return those paths as tuples?

Reply:
You are right, I must have overlooked this at first. Thank you!